llvm/utils/release/build_llvm_package.bat

```diff
@@ -1,134 +1,134 @@
-@echo off

```

Comment:
You shouldn't be making this change, I think. It relates to issues with handling CRLFs. @jackastner @Machiry What's the fix for this?

Reply:
```
git rm .gitattributes
git status
git checkout -- .gitattributes
git reset HEAD .gitattributes
```